### PR TITLE
Fix lock/ban global modlog entries with proof

### PR DESCRIPTION
**Pull request by panpawn**

### CODE REVIEW

File: chat-commands.js [hunk 1]
```diff
@@ -1558,7 +1558,8 @@ exports.commands = {
 		this.add('|unlink|hide|' + userid);
 		if (userid !== toId(this.inputUsername)) this.add('|unlink|hide|' + toId(this.inputUsername));
 
-		this.globalModlog("LOCK", targetUser, " by " + user.name + (target ? ": " + target : ""));
+		const globalReason = (target ? `: ${userReason} ${proof}` : ``);
+		this.globalModlog("LOCK", targetUser, ` by ${user.name} ${globalReason}`);
```
Comment (Zarel) [inline chat-commands.js]: no space

Comment (panpawn) [inline chat-commands.js]: ??

Comment (panpawn) [inline chat-commands.js]: Oh, I see

**Comment by panpawn** [comment]
Updated

**Pull request merged**